packages/editor/src/components/block-icon/index.js

```diff
@@ -25,18 +25,25 @@ function renderIcon( icon ) {
 
 export default function BlockIcon( { icon, showColors = false, className } ) {
 	const renderedIcon = renderIcon( icon && icon.src ? icon.src : icon );
-	if ( showColors ) {
-		return (
-			<div
-				style={ {
-					backgroundColor: icon && icon.background,
-					color: icon && icon.foreground,
-				} }
-				className={ classnames( 'editor-block-icon__colors', className ) }
-			>
-				{ renderedIcon }
-			</div>
-		);
+	const style = showColors ? {
```

Comment:
Wasted effort to assign variable when following condition shortcuts early return without using (should be assigned after condition).